### PR TITLE
[FEATURE] Remove sensible data

### LINKED ISSUE #375: Remove setting "show_sensible_data"
There is no way this setting is required. 
Every sensible setting should ALWAYS be hidden!

### PR DESCRIPTION
**Pull request by simonschaufi**
FIXES #375

Changes proposed in this pull request:
- remove sensible data

Reason for this pull request:
- remove sensible data

### CODE REVIEW

**Comment by kevinpapst** [comment]
This will lead to major merge problems with #688 where I worked on that feature already.
Could you add your changes there?

**Comment by simonschaufi** [comment]
oh shit. so this PR was a waste of time?

**Comment by kevinpapst** [comment]
No, but it might be easier to wait until my PR is merged. Then rebase and merge. OK?

File: installer/install.php [hunk 1]
```diff
@@ -360,32 +360,31 @@ function quoteForSql($input) {
 $query = "INSERT INTO `${p}configuration` (`option`, `value`) VALUES ('revision', '" . $kga['revision'] . "');";
 exec_query($query);
 
-exec_query("INSERT INTO `${p}configuration` (`option`,`value`) VALUES ('currency_name','Euro')");
```
Comment (kevinpapst) [inline installer/install.php:363]: Huh?

Comment (simonschaufi) [reply installer/install.php:363]: no changes, just formatting ;)

**Pull request merged**